packages/babel-generator/src/whitespace.js

```diff
@@ -36,9 +41,14 @@ export default class Whitespace {
     let endToken;
     const tokens = this.tokens;
 
-    let index = this._findToken((token) => token.end - node.end, 0, tokens.length);
+    let index = this._findToken(
+      token => token.end - node.end,
+      0,
+      tokens.length
+    );
     if (index >= 0) {
-      while (index && node.end === tokens[index - 1].end) --index;
+      while (index && node.end === tokens[index - 1].end)
```

Comment:
Can we combine this with ESLint to force these to be wrapped in a block statement?